src/sql/error_message.py

```diff
@@ -0,0 +1,38 @@
+import sqlglot
+
+from ploomber_core.exceptions import COMMUNITY
+
+
+def detail(msg, query=None):
+    msg = str(msg)
+    return_msg = "\nLooks like there is some syntax error " "in your query\n"
```

Comment:
```suggestion
    return_msg = "\nLooks like there is some syntax error " " in your query\n"
```

Also, do we have a test case to cover this?

Reply:
yes in `test_error_messages.py` and `test_postgreSQL.py`

Reply:
Oh I mean to test the string: `"\nLooks like there is a syntax error " " in your query\n"` in the `capsys.readouterr()`

Reply:
No, just added the error specific strings

Reply:
Made the changes